### DISCUSSION TITLE
How to use things in two queries with GroupCompound

### DISCUSSION THREAD

**Discussion by Imitater967** [sebas77/Svelto.ECS#119] [Q&A] [answered]
this is an example 
![image](https://github.com/sebas77/Svelto.ECS/assets/104628217/babfbed7-d41f-4cfb-ba9e-1584722de3a5)

**Comment by Imitater967** [comment]
if not using group compound, this will works like this 
![image](https://github.com/sebas77/Svelto.ECS/assets/104628217/db704882-c3be-4b4c-a76d-d8a713f5d609)

**Reply by Imitater967** [reply]
![image](https://github.com/sebas77/Svelto.ECS/assets/104628217/0a190221-348b-4cb3-8772-edc2104bda32)
this won't work if using groupCompound

**Reply by sebas77** [reply, marked as answer]
I forgot to reply to this, is still a valid question?

**Reply by Imitater967** [reply]
you replied on discord thank you
